ql/ql/src/codeql_ql/style/AcronymsShouldBeCamelCaseQuery.qll

```diff
@@ -29,6 +29,24 @@ string getName(AstNode node, string kind) {
   result = node.(Module).getName() and kind = "module"
 }
 
+string prettyPluralKind(string kind) {
+  kind = "class" and result = "classes"
+  or
+  kind = "classlessPredicate" and result = "predicates"
+  or
+  kind = "classPredicate" and result = "class predicates"
+  or
+  kind = "newtypeBranch" and result = "newtype branches"
+  or
+  kind = "newtype" and result = "newtypes"
```

Comment:
QL already enforces strict case conventions for these declarations. But I suppose there's no harm in still listing them here.

Reply:
I reused the code I already made for detecting name declarations.  
And by doing it this way I'm sure I'm don't miss anything, independently of the conventions enforced by the compiler.